db/schema.rb

```diff
@@ -45,6 +45,15 @@
   add_index "accounts", ["reset_password_token"], name: "index_accounts_on_reset_password_token", unique: true, using: :btree
   add_index "accounts", ["updated_at"], name: "index_accounts_on_updated_at", using: :btree
 
+  create_table "checkins", force: :cascade do |t|
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+  end
+
+  create_table "log_counts", force: :cascade do |t|
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+  end
```

Comment:
Please drop your database and re-migrate it up so it stops adding this to the schema.